worker-example/worker-example-container/configuration/marathon-template-json/marathon-example-worker.json

```diff
@@ -72,7 +72,6 @@
     "${marathon-uris-root}/${worker-config-location}/cfg_${marathon-group}_example_FileSystemDataStoreConfiguration",
     "${marathon-uris-root}/${worker-config-location}/cfg_${marathon-group}_example_ExampleWorkerConfiguration",
     "${marathon-uris-root}/${worker-config-location}/cfg_${marathon-group}_example_RabbitWorkerQueueConfiguration",
-    "${marathon-uris-root}/${worker-config-location}/cfg_${marathon-group}_example_StorageServiceDataStoreConfiguration",
     "${marathon-uris-root}/${docker-login-config}"
```

Comment:
Remove the CAF Storage property placeholders from within this file and remove those properties from here too: https://github.com/WorkerFramework/worker-framework/blob/6a372efe834667671e471d4226149a0667c0040c/worker-example/worker-example-container/configuration/marathon-properties.json